components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -518,44 +518,44 @@ bool PublisherInfoDatabase::GetActivityList(
                       "ON ai.publisher_id = pi.publisher_id "
                       "WHERE 1 = 1";
 
-  if (!filter.id.empty()) {
+  if (!filter->id.empty()) {
```

Comment:
need filter null check in this if `if (!initialized) {`

Reply:
fixed